corehq/apps/es/aggregations.py

```diff
@@ -583,36 +582,19 @@ class NestedTermAggregationsHelper(object):
             ]
         ).get_data()
 
-        # summing the balances of ledger values, grouped by the entry id
-
-        NestedTermAggregationsHelper(
-            base_query=LedgerES().domain(domain).section(section_id),
-            terms=[
-                AggregationTerm('entry_id', 'entry_id'),
-            ],
-            inner_most_aggregation=SumAggregation('balance', 'balance'),
-        ).get_data()
```

Comment:
this isn't a ledger specific feature right? Is it just that we never use this outside of ledgers?